PsimagLite/src/tests/CMakeLists.txt

```
@@ -0,0 +1,3 @@
+add_executable(test_AkimaSpline test_AkimaSpline.cpp)
+target_link_libraries(test_AkimaSpline PRIVATE Catch2::Catch2WithMain psimaglite)
```

We want to always refer to the qualified library so that in principle we could use an external PsimagLite
```suggestion
target_link_libraries(test_AkimaSpline PRIVATE Catch2::Catch2WithMain psimaglite::psimaglite)
```